www/cgi-bin/minerinfo

```diff
@@ -50,6 +50,7 @@ LSB_RELEASE=$(lsb_release -s -d)
 UNAME=$(uname -r)
 SYSTEM_BOOT_TIME=$(uptime -s)
 SYSTEM_UP_TIME=$(uptime -p)
+INTERNET_IP=$(curl smart-ip.net/myip)
```

Comment:
A lookup request like:
`dig +short myip.opendns.com @resolver1.opendns.com`
could be faster and does not depend on curl, please evaluate the differences.
Also, make sure minerinfo loads correctly in case no internet connection is available.